Mathlib/Topology/Category/TopCat/Basic.lean

```diff
@@ -48,6 +49,7 @@ instance concreteCategory : ConcreteCategory TopCat := by
   dsimp [TopCat]
   infer_instance
 
+@[to_additive existing TopCat.instCoeSortTopCatType]
 instance : CoeSort TopCat (Type*) :=
```

Comment:
Given that the name is used explicitly in `to_additive` I think it's worth naming this:
```suggestion
instance instCoeSortTopCatType : CoeSort TopCat (Type*) :=
```
(and anyway, instance naming is a Good Thing, especially in Lean 4).